guides/v2.1/config-guide/cli/config-cli-subcommands-static-view.md

```diff
@@ -51,108 +51,401 @@ Command options:
 The following table explains this command's parameters and values.
 
 <table>
-	<col width="15%" />
-  	<col width="65%" />
-  	<col width="25%" />
-	<tbody>
-		<tr>
-			<th>Option</th>
-			<th>Description</th>
-			<th>Required?</th>
-		</tr>
-	<tr>
-        <td>&lt;languages&gt;</td>
-        <td><p>Space-separated list of language codes to output static view files. (Default is <code>en_US</code>.)</p>
-        <p>You can find the list by running <code>magento info:language:list</code>.</p></td>
-    <td><p>No</p></td>
+  <col width="25%" />
+  <col width="60%" />
+  <col width="10%" />
+  <col width="10%" />
+  <tbody>
+    <tr>

```

Comment:
Can we have the `tr` and `th`  when is short `inline`:
e:g 
```
<tr>
	<th>Option</th>
	<th>Description</th>
	<th>Required?</th>
</tr>
```
We will increase the readbility. This comment should apply in all PR.

Please review it

Reply:
@osrecio I have updated table structure view. Please check it.